content/terraform-docs-agents/v1.22.x/docs/cloud-docs/agents/agents.mdx

```diff
@@ -247,7 +247,6 @@ After initiating a graceful shutdown by either of these methods, the terminal us
 
     Environment variable: `TFC_AGENT_ACCEPT`
 
-@include 'policy.mdx'
```

Comment:
idk why this automatically removed itself but i think we wanna keep it

Reply:
That screenshot actually shows that the text is in here just below the include. Seems like it is removing a dup of content and we should keep this.

Reply:
it was recently added so i think if you wanna remove it you're gonna need to talk to the folks who added it first https://github.com/hashicorp/web-unified-docs-internal/pull/96